<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>701</width>
      <height>37</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Transfer Type : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="TransferTypeComboBox">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Broadcast</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Receiver</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Clients List :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ClientsListComboBox">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="SendFilePushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>440</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <italic>true</italic>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Send FIle</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditMessages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
